### PR TITLE
Upgrade backend

### PR DESCRIPTION
**Pull request by Hrabarchuk**

### CODE REVIEW

**Review by Karonuk — APPROVED** [review]

**Review by bodkia22 — APPROVED** [review]

File: src/YPS.WebUI/Startup.cs [hunk 1]
```diff
@@ -60,7 +64,14 @@ public void ConfigureServices(IServiceCollection services)
...
                 });
-              
+
+                var xmlFile = $"{Assembly.GetExecutingAssembly().GetName().Name}.xml";
+                var xmlPath = Path.Combine(AppContext.BaseDirectory, xmlFile);
+                if (File.Exists(xmlPath))
+                {
+                    c.IncludeXmlComments(xmlPath);
+                }
+
```
Comment (DmMov) [inline src/YPS.WebUI/Startup.cs:74]: What is this for?

File: src/YPS.WebUI/Controllers/AuthController.cs [hunk 2]
```diff
@@ -16,12 +17,34 @@ public AuthController(IConfiguration config)
...
+                return Ok(viewModel);
+            }
+            catch (ValidationException e)
+            {
+                return BadRequest(e.Message);
+            }
+            catch (Exception)
+            {
+                return BadRequest("Invalid password ");
+            }
```
Comment (DmMov) [inline src/YPS.WebUI/Controllers/AuthController.cs:47]: Why is it here?

File: src/YPS.WebUI/Controllers/AuthController.cs [hunk 3]
```diff
@@ -16,12 +17,34 @@ public AuthController(IConfiguration config)
...
             _apiKey = config.GetValue<string>("ApiKey");
         }
+
+        /// <summary>
+        /// Login
+        /// </summary>
+        /// <param name="command"></param>
+        /// <response code="200">Returns user token</response>
+        /// <response code="401">Invalid username or password</response>
+        /// <returns></returns>
```
Comment (DmMov) [inline src/YPS.WebUI/Controllers/AuthController.cs:27]: Who wrote this coment?

**Comment by bodkia22** [comment]
test

**Pull request closed without merging**